edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -43,6 +43,8 @@ public static class Constants
 
         public const string EdgeModuleHubServerCertificateFileKey = "EdgeModuleHubServerCertificateFile";
 
+        public const string DisableSeparePullFromCreateModule = "DisableSeparePullFromCreateModule";
```

Comment:
Can we call this CheckImagePullBeforeModuleCreate or EnableImagePullBeforeModuleCreate
Also there is a typo in the spelling - DisableSepar**at**ePullFromCreateModule 

(Naming is hard!!)

Reply:
Done!